controllers/patientsController.ts

```diff
@@ -152,7 +152,22 @@ const create = async (req: Request, res: Response): Promise<void> => {
   const isStudentStatusVerified = Boolean(req.body.isStudentStatusVerified);
   const hasPrescription = Boolean(req.body.hasPrescription);
 
+  const today = new Date();
+  const hundredYear = new Date(today.setFullYear(today.getFullYear() - 100));
+  const newToday = new Date();
+  const tenYear = new Date(newToday.setFullYear(newToday.getFullYear() - 10));
+
   const psychologistId = req.user.psychologist;
+  let infoMessage = `L'étudiant ${firstNames} ${lastName} a bien été créé.`;
+
+  if (dateOfBirth && dateOfBirth.getFullYear() < hundredYear.getFullYear()) {
+    infoMessage += 'Votre étudiant ne peut avoir plus de 100 ans';
+    throw new CustomError('Votre étudiant ne peut avoir plus de 100 ans', 400);
+  }
+  if (dateOfBirth && dateOfBirth.getFullYear() > tenYear.getFullYear()) {
+    infoMessage += 'Votre étudiant ne peut avoir moins de 10 ans';
```

Comment:
What's the point of doing that, the info message is never send because you are throwing an exception

Reply:
I delete it.